src/app/recipe-list/recipe-list.component.html

```diff
@@ -20,29 +20,32 @@
     </a>
   </div>
   <div class="recipes__list">
-    <ng-container *ngIf="(recipes$ | async) as recipes; else loading">
-      <app-recipe-tile
-        *ngFor="let recipe of recipes"
-        [recipe]="recipe"
-        (recipeSelected)="fetchRecipe($event)"
-        (recipeDeleted)="deleteRecipe($event)"
-      ></app-recipe-tile>
-      <div
-        *ngIf="!recipes.length"
-        class="no-recipes"
-      >
-        <mat-icon class="icon--large faded">
-          indeterminate_check_box
-        </mat-icon>
-        <span class="text--large faded">
-          No recipes found...
-        </span>
-      </div>
+    <ng-container *ngIf="(loading$ | async) === false">
+      <ng-container *ngIf="recipes$ | async as recipes">
+        <app-recipe-tile
+          *ngFor="let recipe of recipes$ | async"
```

Comment:
możesz dać po prostu recipe of recipes bez asynca, bo masz to samo dwie linijki wyżej i w ten sposób tworzysz drugą subskrypcję